source/components/_three-blocks.erb

```diff
@@ -0,0 +1,17 @@
+<div class="govuk-o-grid">
```

Comment:
Sorry, I know I've already reviewed this but looking again… this isn't really a _component_ – it's a _specific_ part of the homepage that's not reusable, and it just uses the existing grid classes.

Especially as it's not being re-used across multiple files, I feel like this should either just be part of the index.html.erb file, or it should be in a different folder (i.e. not components).

I also think that's why it's name is a little weird - it's really just a part of the homepage so if it was going to continue to be a separate file maybe `homepage_lower` or `homepage_section_list` or something like that would be a better name?